sleap_nn/training/model_trainer.py

```
@@ -53,6 +54,7 @@
     CentroidStreamingDataset,
     SingleInstanceStreamingDataset,
 )
+from sleap_nn.training import get_bin_files
```

Ruff

`sleap_nn.training.get_bin_files` imported but unused

Remove unused import: `sleap_nn.training.get_bin_files`

(F401)